config/config.go

```diff
@@ -0,0 +1,35 @@
+package config
+import(
+	"strings"
+	"github.com/spf13/viper"
+)
+type Qiniu struct {
+	AccessKey string
+	SecretKey string
+	Bucket    string
+	Domain    string
+}
+var Q Qiniu

```

Comment:
The global variable `Q` is ambiguous.

Maybe add a new type `Config` and a global variable `Conf` is better, IMO.